includes/footer.html

```diff
@@ -60,7 +60,12 @@ <h5 class="white-text fine">coala</h5>
     </div>
     <div class="footer-copyright">
         <div class="container">
-            <span class="left">© 2017 coala</span>
+            <span class="left">© 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpr9xhq323/includes/footer.html
+++ b/tmp/tmpr9xhq323/includes/footer.html
@@ -60,7 +60,7 @@
     </div>
     <div class="footer-copyright">
         <div class="container">
-            <span class="left">© 
+            <span class="left">©
                 <script>
                     document.write(new Date().getFullYear());
                 </script> 
```